server/src/main/java/io/crate/plugin/SQLPlugin.java

```diff
@@ -195,14 +191,9 @@ public SQLPlugin(Settings settings) {
         modules.add(new OperatorModule());
         modules.add(new PredicateModule());
         modules.add(new MonitorModule());
-        modules.add(new AggregationImplModule());
-        modules.add(new ScalarFunctionModule());
```

Comment:
These moved to `Node` because `NodeContext` is now used in `TransportCreateIndexAction` which is also initialized in cases where the `SQLPlugin` is not loaded. (By tests using `ESIntegTestCase`)